src/main/java/client/Client.java

```diff
@@ -0,0 +1,29 @@
+package main.java.client;
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+
+public class Client {
+
+    private static final int PORT = 8080;
+    private static final String HOST = "127.0.0.1";
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this IP "127.0.0.1" address configurable. ([squid:S1313](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AS1313))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHTTPServer&pullRequest=1&issues=AWQTU_q-bXXTsx7O43dV&open=AWQTU_q-bXXTsx7O43dV)

[ISSUE_KEY]: AWQTU_q-bXXTsx7O43dV